src/client.js

```diff
@@ -0,0 +1,9 @@
+import { is } from 'ramda'
+
+export function browserLanguage (navigatorLanguage = 'cs') {
+  if (is(String, navigatorLanguage) && navigatorLanguage.toLowerCase().startsWith('cs')) {
```

Comment:
is() check is redundant,. in case navigatorLanguage is undefined, it falls back to the default parameter.